code/__DEFINES/_cooldowns.dm

```diff
@@ -62,6 +62,14 @@
 
 #define COOLDOWN_TIMELEFT(cd_source, cd_index) (max(0, cd_source.cd_##cd_index - world.time))
 
+// kevin...
+// the cd indexes arent supposed to start with cd_
+#define COOLDOWN_START_A(cd_source, cd_index, cd_time) (cd_source.cd_index = world.time + (cd_time))
```

Comment:
this seems to do the exact same thing as `COOLDOWN_START` and `COOLDOWN_FINISHED`

also i started using `CD_INDEX_` for specifically timer indexes because i want to namespace defines where it makes sense to; there's just so many defines floating around the codebase that typing anything in uppercase gives you a ton of suggestions from different things

having grouping helps a lot (e.g. i don't have to guess if `ATOM_INITAILIZED` or `ATOM_FLAG_INITIALIZED` if we use a verbose version is an atom flag, whereas `NOCONDUCT` and `NOBLOODY` tells me absolutely nothing in the name. this is what leads to people putting clothing flags on item flags and vice versa, because we didn't use to have very obvious tells *in the name* that it's improper)

this does make devs have to type a bit more but i think it's worth it for long term making understanding easier